service/worker/archiver/archiver.go

```diff
@@ -103,12 +103,12 @@ func handleRequest(ctx workflow.Context, logger log.Logger, metricsClient metric
 	sw := metricsClient.StartTimer(metrics.ArchiverScope, metrics.ArchiverHandleRequestLatency)
 	logger = tagLoggerWithRequest(logger, request)
 	ao := workflow.ActivityOptions{
-		ScheduleToStartTimeout: 10 * time.Minute,
-		StartToCloseTimeout:    5 * time.Minute,
+		ScheduleToStartTimeout: 2 * time.Minute,
+		StartToCloseTimeout:    2 * time.Minute,
 		RetryPolicy: &cadence.RetryPolicy{
 			InitialInterval:          time.Second,
 			BackoffCoefficient:       2.0,
-			ExpirationInterval:       10 * time.Minute,
+			ExpirationInterval:       4 * time.Minute,
```

Comment:
Seems fine, we can tune this if we find a need. But I think this is plenty long.

Reply:
If we want to use a lower value, we also need to decrease schedule to start timeout.